sdk/search/search-documents/samples-dev/setup.ts

```diff
@@ -248,7 +248,8 @@ export async function createIndex(client: SearchIndexClient, name: string): Prom
       allowedOrigins: ["*"],
     },
     vectorSearch: {
-      algorithmConfigurations: [{ name: "vector-search-configuration", kind: "hnsw" }],
+      algorithms: [{ name: "vector-search-algorithm", kind: "hnsw" }],
+      profiles: [{ name: "vector-search-profile", algorithm: "vector-search-algorithm" }],
```

Comment:
Do we want to include any `vectorizers` in this setup?

Reply:
Done!